src/gc/transaction_level_gc_manager.cpp

```diff
@@ -96,18 +100,19 @@ int TransactionLevelGCManager::Unlink(const int &thread_id,
 
   // First iterate the local unlink queue
   local_unlink_queues_[thread_id].remove_if(
```

Comment:
The function for `local_unlink_queues_` is that if an item in `unlink_queue_` is not garbage for now, that item should be put back again for the future gc. The reason you separate `local_unlink_queues_` from `unlink_queue_` is that `unlink_queue_` uses `LockFreeQueue`, but `local_unlink_queues_` uses `std::List`. So why `unlink_queue_` can not use  `std::List` as well, then we don't need `local_unlink_queues_`?